transcription/transcription.go

```diff
@@ -0,0 +1,31 @@
+package transcription
+
+import (
+	"net/smtp"
+	"strings"
+)
+
+// SendEmail authenticates, creates message, and sends email using arguments.
+// Host is email server, e.g. "smtp.gmail.com". Port of server required, e.g. 25.
+func SendEmail(username string, password string, host string, port int, to []string, subject string, body string) error {
+	from := username
+	auth := smtp.PlainAuth("", username, password, host)
+
+	// The msg parameter should be an RFC 822-style email with headers first,
+	// a blank line, and then the message body. The lines of msg should be CRLF terminated.
+	msg := []byte(msgHeaders(from, to, subject) + "\r\n" + body + "\r\n")
+	addr := host + ":" + string(port)
+	if err := smtp.SendMail(addr, auth, from, to, msg); err != nil {
+		return err
+	}
+	return nil
+}
+
+// The msg headers should usually include fields such as "From", "To", "Subject", and "Cc".
```

Comment:
function comments should always starting with the name of the function.

How about "msgHeaders includes fields such as..."?

Also, is this comment necessary? Would someone reading the code not understand what msgHeaders does?